apollo-router/src/compute_job.rs

```diff
@@ -14,13 +15,28 @@ use crate::metrics::meter_provider;
 /// reaches `QUEUE_SOFT_CAPACITY_PER_THREAD * thread_pool_size()`
 const QUEUE_SOFT_CAPACITY_PER_THREAD: usize = 20;
 
+static CONFIGURED_POOL_SIZE: AtomicUsize = AtomicUsize::new(0);
+
+pub(crate) fn experimental_set_thread_pool_size(size: usize) {
+    CONFIGURED_POOL_SIZE.store(size, std::sync::atomic::Ordering::Release);
+}
+
 /// Let this thread pool use all available resources if it can.
 /// In the worst case, we’ll have moderate context switching cost
 /// as the kernel’s scheduler distributes time to it or Tokio or other threads.
```

Comment:
Did you want to update this comment in this PR?

Reply:
I tested this manually and it appears to be working as intended. Thanks!

Reply:
Ha, good spot :) Thanks for testing it out, and it's great to know that it has the intended effect. I neglected to update code comments and the like.
We'll need to figure out if we want to put this configuration option in a release (with testing & docs & a resizeable pool to support hot reload) or if we want to do something else (like using a non-configurable formula to set a sensible max pool size) sometime in the next 10 days or so.